### PR TITLE
add tile support

### PR DESCRIPTION
**Pull request by gkdwoe**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -8,18 +8,22 @@ int main(int argc, char** argv, char** envp) {
...
     bool        noFractional = false;
+    bool        tileWallpaper = false;
     for (int i = 1; i < argc; ++i) {
         if ((!strcmp(argv[i], "-c") || !strcmp(argv[i], "--config")) && argc >= i + 2) {
             configPath = std::string(argv[++i]);
             Debug::log(LOG, "Using config location %s.", configPath.c_str());
         } else if (!strcmp(argv[i], "--no-fractional") || !strcmp(argv[i], "-n")) {
             noFractional = true;
             Debug::log(LOG, "Disabling fractional scaling support!");
+        } else if (!strcmp(argv[i], "--tile") || !strcmp(argv[i], "-t")) {
```
Comment (vaxerski) [inline src/main.cpp:19]: ?!?! add this to the config not as a flag, just like contain

**Review by gkdwoe — COMMENTED** [review]
updated to use config instead of flag

**Review by gkdwoe — COMMENTED** [review]
ok ok i think i got it this time..

File: src/config/ConfigManager.cpp [hunk 2]
```diff
@@ -141,6 +150,10 @@ static Hyprlang::CParseResult handleReload(const char* C, const char* V) {
         WALLPAPER = WALLPAPER.substr(8);
     }
 
+    if (WALLPAPER.find("tile:") == 0) {
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:153]: no {}

**Review by vaxerski — COMMENTED** [review]
rest lgtm wiki mr needed

File: src/config/ConfigManager.cpp (hunk 2 above)
Comment (gkdwoe) [reply src/config/ConfigManager.cpp]: submitted something to wiki

**Review by vaxerski — APPROVED** [review]
thanks!

**Pull request merged**